docker/docker-compose.test.yml

```diff
@@ -13,6 +13,17 @@ services:
     command: postgres -F
     network_mode: bridge
 
+  musicbrainz_db:
+    image: metabrainz/musicbrainz-test-database:beta
+    volumes:
+      - ../data/mbdata:/var/lib/postgresql/data/pgdata:z
```

Comment:
Not necessary in a test environment.

Reply:
Tests fail without the volumes part.

Reply:
Could you please share the complete log when you run the tests?

Reply:
https://gist.github.com/spellew/ec08900e4f5d8ddafd02bf1374723ad4

Reply:
Well, setting up the volume works because it picks up the schema and the database from the host volume, which contains the database from your dev environment. I am re-pulling the `musicbrainz-test:beta` image to see if there is some problem with the current image.

Reply:
So, I re-pulled and checked the image again, there seems to be a problem with `musicbrainz-test-database` image. I'll take a look into it, and why it fails. (maybe some recent code changes)

Reply:
Tests still seem to be failing.

Reply:
As you should be able to see in Jenkins build. 
```
jenkinsbuildjenkinscritiquebrainzpytest1096_critiquebrainz_redis_1
2019/07/03 22:51:20 Waiting for host: tcp://db_test:5432
2019/07/03 22:51:30 Connected to tcp://db_test:5432
2019/07/03 22:51:30 Waiting for host: tcp://musicbrainz_db:5432
2019/07/03 22:52:30 Timeout after 1m0s waiting on dependencies to become available: [tcp://musicbrainz_db:5432]
2019/07/03 22:52:30 Command exited with error: exit status 1
Error response from daemon: Could not find the file /data/test_report.xml in container jenkinsbuildjenkinscritiquebrainzpytest1096_critiquebrainz_test_1
Stopping jenkinsbuildjenkinscritiquebrainzpytest1096_db_test_1              ... 
Stopping jenkinsbuildjenkinscritiquebrainzpytest1096_musicbrainz_db_1       ... 
```
Try increasing the Timeout to 120 seconds, and if not to 180 seconds or more. Thanks! :)

Reply:
(timeout in dockerize)

Reply:
Tests pass. Awesome @spellew!! \o/

Reply:
Thank you for your help! :) @ferbncode